sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Average.scala

```diff
@@ -52,11 +51,13 @@ case class Average(child: Expression) extends DeclarativeAggregate with Implicit
   private lazy val resultType = child.dataType match {
     case DecimalType.Fixed(p, s) =>
       DecimalType.bounded(p + 4, s + 4)
+    case CalendarIntervalType => CalendarIntervalType
```

Comment:
nit: `case interval: CalendarIntervalType => interval`?

Reply:
ok, I will change this and the one in the sumDataType